kinto/core/storage/postgresql/schema.sql

```diff
@@ -23,9 +23,9 @@ IMMUTABLE;
 -- Actual records
 --
 CREATE TABLE IF NOT EXISTS records (
-    id TEXT NOT NULL,
-    parent_id TEXT NOT NULL,
-    collection_id TEXT NOT NULL,
+    id TEXT COLLATE "C" NOT NULL,
```

Comment:
The comment in `migration_018_019.sql` about *why* the use of a specific collation would be useful here.